registry/debug-adapters.yml

```diff
@@ -1,101 +1,474 @@
 debug-adapters:
   - name: "CMSIS-DAP@pyOCD"
-    alias-name: ["CMSIS-DAP", "DAP-Link"]     # alternative names that map to this debug adapter
-    template: CMSIS-DAP-pyOCD.adapter.json    # template file
-    gdbserver:                                # add gdbserver: node under debugger: in cbuild-run.yml
-    defaults:                                 # default values to use when nowhere specified
-      port: 3333                              # default value of first gdbserver port
+    alias-name: ["CMSIS-DAP", "DAP-Link"]       # alternative names that map to this debug adapter
+    template: CMSIS-DAP-pyOCD.adapter.json      # template files
+    gdbserver:                                  # add gdbserver: node under debugger: in cbuild-run.yml
+    defaults:                                   # default values to use when nowhere specified
+      port: 3333                                # default value of first gdbserver port
       protocol: swd
       clock: 10000000
+      trace:
+        trace-port: SWO-UART
+    ui:
+      - section: Debug Interface
+        description: Interface configuration for the debug port
+        options:
+          - name: Clock (kHz)
+            description: Trace configuration
```

Comment:
This description seems to be in the wrong place.
See more similar occurrences below.
```suggestion
```